packages/tools/examples/advancedMagnifyTool/index.ts

```diff
@@ -174,10 +174,12 @@ async function addSegmentationsToState(volumeId: string) {
   }
 
   // Create a segmentation of the same resolution as the source data
-  segmentationVolume =
-    await volumeLoader.createAndCacheDerivedSegmentationVolume(volumeId, {
+  segmentationVolume = await volumeLoader.createAndCacheDerivedLabelmapVolume(
```

Comment:
What about allowing the second parameter to be either a string or the full definition?  It strikes me that most calls will just pass a string in, and it is much more obvious to me to be 
await volumeLoader.createAndCacheDerivedLabelmapVolume(volumeId,segmentationid)
for the 90% use case.

Reply:
The library evolves over time, and we may eventually need options for various reasons I'm pretty sure. I'm confident we'll regret not including them.

Reply:
I'm not saying to now allow the full definition, just that it is easy enough to support a string version of it for the 90% use case of only specifying segmentation id.  In fact one already needs other parameters as an option, just that making it easy to create it is convenient and is easier to read in the code.